engine/lib/phx/src/audio/audio.rs

```diff
@@ -90,12 +91,19 @@ impl Audio {
         self.audio_manager.num_sounds() as u64
     }
 
-    pub fn get_playing_count(&self) -> u64 {
-        self.sounds
-            .iter()
-            .filter(|sound| sound.borrow().state() == PlaybackState::Playing)
-            .count() as u64
-    }
+    //pub fn get_playing_count(&self) -> u64 {
```

Comment:
Do we still need this? I think it should be on the Lua side now.

Reply:
that´s correct :)